### ISSUE TITLE
[Enhancement] releases Releases on Repository of Chocolatey packages

### ISSUE THREAD

**Issue by 4-FLOSS-Free-Libre-Open-Source-Software** [xatupal/DarkTheme#3]
Hello, this is not an issue in itself. Just an suggestion. I see you have in your repo 3 Plugins today already. Would be nice to have a auto update feature for this primary security applications plugins from trusted source. The sha hash of PLRGX can be verified by installation automatically optionally if specified.  _For the lazy ones :) and_ to fix incompatibility issue with future of keepass with plugin by getting all in one place for example.

I don't know if you have the spare time to maintain in parallel the "packages". Which basically would contains a script to URL of github latest release asset. 

If you may want to take a look here if you don't know already:

[keepass packages](https://chocolatey.org/packages?q=keepass&moderatorQueue=&prerelease=true&sortOrder=package-download-count)

There are many plugins and packages count is growing. 

Perhaps for the day, this goes out of "Pre-release"? While in current version so far it works very well anyway.

**Comment by xatupal** [comment]
Thanks for suggestion. Sounds easy. Why not to do it if there exists (or can be created) a single script that will create and publish Chocolatey package.

The plugin is "pre-relase" because there will be a name change. It is now not only a dark theme.

**Comment by xatupal** [comment]
My plugins at [Chocolatey](https://chocolatey.org/profiles/xatupal).
